src/Assets/AAsset.cpp

```diff
@@ -0,0 +1,91 @@
+#include "AAsset.hpp"
+
+AAsset::AAsset()
+{
+}
+
+AAsset::~AAsset()
+{
+}
+
+std::string AAsset::getFileName()
+{
+    return _fileName;
+}
+
+std::string AAsset::getDisplayName()
+{
+    return _displayName;
+}
+
+std::vector<std::string> AAsset::getTags()
+{
+    return _tags;
+}
+
+void AAsset::setFileName(std::string fileName)
+{
+    _fileName = fileName;
+}
+
+void AAsset::setDisplayName(std::string displayName)
+{
+    _displayName = displayName;
+}
+
+void AAsset::setTags(std::vector<std::string> tags)
+{
+    _tags = tags;
+}
+
+void AAsset::addTags(std::vector<std::string> newTags)
+{
+    _tags.insert(_tags.end(), newTags.begin(), newTags.end());
+}
+
+void AAsset::addTag(std::string newTag)
+{
+    _tags.push_back(newTag);
+}
+
```

Comment:
In general it is quite common, for the sake of class readability, to define the constructor/destructor, the getter/setter and generally all functions that take a line of code, directly inline with the function name, in the hpp